test/e2e/README.md

```diff
@@ -0,0 +1,26 @@
+# Cross OS E2E Tests
+
+Tests designed to work with both Linux and Windows workers should use the `operating_system` paramater to differentiate differences between operating systems.
+
+Export the `OPERATING_SYSTEM` environment variable to either "linux" or "windows", depending on the OS of the worker that you would like to test.
+
+# OS Specific Tests
+Use the `mark.skipif` paramater to differentiate between operating systems for os specific tests.
+```
+@pytest.mark.skipif(
+        os.environ["OPERATING_SYSTEM"] == "windows",
+        reason="Linux specific test",
+    )
+
+@pytest.mark.skipif(
+        os.environ["OPERATING_SYSTEM"] == "linux",
+        reason="Linux specific test",
+    )
+```
+
+# Test Scoping
+**Session** - Tests that do not need to perform host or worker configuration, or will not impact the outcome of another test within session scope. Use the `session_worker` fixture in `conftesy.py` to specify this scope.
```

Comment:
It might be good to explain the benefit of using shared session-scoped workers across tests (e.g. wall-clock time associated with starting an instance and setting up the worker)

Reply:
Added.